### ISSUE TITLE
Argument like "%s", "%d" causes broken layout.

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#56]
```
$ xpanes -V; tmux -V
xpanes 2.0.2
tmux 2.2
```

```
$ xpanes '%d' '%s'
```

expected: 2panes
actual: 3panes

It seems that this is caused by `perl`.
`printf` method in `perl` interprets such the query.

**Comment by greymd** [comment]
In addition, some tmux version cannot recognize window name which includes `%`.

```
root@6e9ece0ec585:/work# tmux new-window -n %s-3087 -d
root@6e9ece0ec585:/work# tmux split-window -t %s-3087 -h
can't find pane %s-3087
```

**Comment by greymd** [comment]
Fixed with [v2.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v2.1.0).
